content/_index.md

```diff
@@ -11,6 +11,8 @@ title = "Taiwan Gold Card"
 - Do I qualify for the Gold card?
 
 {{< button "application-faq/" "Access the FAQ" >}}
+{{< button "https://coa.immigration.gov.tw/coa-frontend/four-in-one/entry/golden-card?src=taiwangoldcard.com" "Apply for the Gold Card" >}}
```

Comment:
I don't think this is the right place to put this link. More people will click it than going to the FAQ, and they'll end up on the application portal having read no instructions. Since there's no instructions or information on the application portal, that will be a bad user experience. It's better if they work out how they are qualified and go to the application portal as they read our content on how to apply.